src/otaclient/ota_core/_client_updater.py

```diff
@@ -61,7 +64,6 @@ def _execute_client_update(self):
         logger.info(f"execute local update({ecu_info.ecu_id=}): {self.update_version=}")
 
         try:
-            self._handle_upper_proxy()
```

Comment:
wait for upper proxy(and downloader configuration) is done by the IPC interface(OTAClient) in `_main` module.